homeassistant/components/arcam_fmj/strings.json

```diff
@@ -1,7 +1,29 @@
 {
+    "config": {
+        "abort": {
+            "already_setup": "Device was already setup.",
+            "updated_instance": "Updated instance with new host address."
+        },
+        "error": {},
+        "flow_title": "Arcam FMJ on {host}",
+        "step": {
+            "confirm": {
+                "description": "Do you want to add Arcam FMJ on `{host}` to Home Assistant?",
+                "title": "Discovered Arcam FMJ"
+            },
+            "user": {
+                "data": {
+                    "host": "[%key:common::config_flow::data::host%]",
+                    "port": "[%key:common::config_flow::data::port%]"
+                },
+                "description": "Please enter the host name or IP address of device.",
+                "title": "Connect to Arcam FMJ"
```

Comment:
Consider dropping this. It will be automatically filled in with the manifest name if omitted.

Reply:
For steps too?

Reply:
Yes